### PR TITLE
enhances project types activity log events

### PR DESCRIPTION
**Pull request by tillyw**
this pr refines project types activity log events using REST connectors.

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/11310

## Testing
**URL to test:** 

please test locally as this pr is comparing against another activity log branch

**Steps to test:**
- add, edit and remove a project type while noting the activity log updates
- the changes should be reflected in the activity log

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by johnclary — CHANGES_REQUESTED** [review]
@tillyw this works exactly as promised :) :)


would you swap out the `BeenHere` icon for this `summarize` icon? That's the icon we use for misc project edits, like description?

```jsx
 <span className="material-symbols-outlined">summarize</span>
```

Screen Shot 2023-01-27 at 10 54 17 AM

File: moped-editor/src/utils/activityLogFormatters/mopedProjectTypesActivity.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import BeenhereOutlinedIcon from "@material-ui/icons/BeenhereOutlined";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatProjectTypesActivity = (change, projectTypeList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_project_types"];
```
Comment (mddilley) [inline moped-editor/src/utils/activityLogFormatters/mopedProjectTypesActivity.js:5]: Ran this locally and noticed there is a compile warning that this var is never used. Might be used down the line but wanted to note that

File: moped-editor/src/utils/activityLogFormatters/mopedProjectTypesActivity.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      changeIcon,
+      changeText: [
+        { text: "Added ", style: null },
+        displayText,
+        { text: " as a project type ", style: null },
+      ],
+    };
+  }
+
+  // delete an existing component
```
Comment (mddilley) [inline moped-editor/src/utils/activityLogFormatters/mopedProjectTypesActivity.js:27]: Small change to this comment since this is for types

**Review by mddilley — DISMISSED** [review]
Tested locally, and this works beautifully! 👏 🚀 🎉  Gtg after the icon change that John requested

**Comment by tillyw** [comment]
thanks, y'all, made these changes!

**Review by johnclary — APPROVED** [review]
🚢 🙌

**Pull request merged**